include/nbind/signature/BaseSignature.h

```diff
@@ -178,7 +178,10 @@ class TemplatedBaseSignature : public BaseSignature {
 
 		if(!arityIsValid(nanArgs)) {
 			// TODO: When function is overloaded, this test could be skipped...
-			Nan::ThrowError("Wrong number of arguments");
+
+			char buffer [40];
+			sprintf (buffer, "Wrong number of arguments, expected %d", (sizeof...(Args)));
```

Comment:
`snprintf` is safer, but not portable. We could use C++11 libraries:

```c++
std::string msg = "Wrong number of arguments, expected " + std::to_string(sizeof...(Args));
```

Then either `Nan::ThrowError(msg)` or `Nan::ThrowError(msg).c_str()` (if the other one doesn't work).